### DISCUSSION TITLE
Outfits

### DISCUSSION THREAD

**Discussion by vcMoo** [YimMenu/YimMenu#1781] [General]
Does anyone have any nice female outfits 
Please sends codes if you do im in need of some outfits :p

**Comment by iDominiqe** [comment]
[saved_outfits.zip](https://github.com/YimMenu/YimMenu/files/12114432/saved_outfits.zip)

**Reply by vcMoo** [reply]
tyyyy

**Reply by AZGamer044** [reply]
do you have some male modded outfits for yim too?
